lib/zendesk_apps_support/package.rb

```diff
@@ -190,7 +190,7 @@ def has_location?
     deprecate :has_location?, 'manifest.location?', 2016, 9
 
     def has_file?(path)
-      File.exist?(path_to(path))
+      File.file?(path_to(path))
```

Comment:
No, that wouldn't work (and even if it did, we never documented it as supported).